### ISSUE TITLE
The UI is a little ugly, asking for a nice way to interact

### ISSUE THREAD

**Issue by qinyuanqiblog** [spmbt/googleSearchExtraButtons#13]
Thank you very much for your developed so useful plugin, but I'm in the process of use, found that the UI is a bit ugly, inconvenient operation, so I hope you'll take time to change in the free under the UI. I come from China, English is not very good, is now talking with you through the translation software, please forgive me.

**Comment by spmbt** [comment]
Yes, I will accept suggestions for introducing an additional style of design or for fixing the current one and will insert it into the script (pictures or the desired change of color codes in the existing design is enough).

**Comment by mikhoul** [comment]
@spmbt  In few weeks I will try to take my time to make some suggestions to have a better UI with some mockups. 😃 

In meantime one improvement I'd like to see is to have 2 more options for the search:

**1 -** Being able to choose the country from where the results come, if I want results from UK it would ad at the end of the search string  `&gl=uk` .

**2-** Being able to specify in what language we want the searach results, it's done with `&lr=lang_fr` for the french.

Here's a table for the search Language Codes:

lr=lang_af    Afrikaans
lr=lang_ar    Arabic
lr=lang_hy    Armenian
lr=lang_be    Belarusian
lr=lang_bg    Bulgarian
lr=lang_ca    Catalan
lr=lang_zh-CN Chinese (Simplified)
lr=lang_zh-TW Chinese (Traditional)
lr=lang_hr    Croatian
lr=lang_cs    Czech
lr=lang_da    Danish
lr=lang_nl    Dutch
lr=lang_en    English
lr=lang_eo    Esperanto
lr=lang_et    Estonian
lr=lang_tl    Filipino
lr=lang_fi    Finnish
lr=lang_fr    French
lr=lang_de    German
lr=lang_el    Greek
lr=lang_iw    Hebrew
lr=lang_hi    Hindi
lr=lang_hu    Hungarian
lr=lang_is    Icelandic
lr=lang_id    Indonesian
lr=lang_it    Italian
lr=lang_ja    Japanese
lr=lang_ko    Korean
lr=lang_lv    Latvian
lr=lang_lt    Lithuanian
lr=lang_no    Norwegian
lr=lang_fa    Persian
lr=lang_pl    Polish
lr=lang_pt    Portuguese
lr=lang_ro    Romanian
lr=lang_ru    Russian
lr=lang_sr    Serbian
lr=lang_sk    Slovak
lr=lang_sl    Slovenian
lr=lang_es    Spanish
lr=lang_sw    Swahili
lr=lang_sv    Swedish
lr=lang_th    Thai
lr=lang_tr    Turkish
lr=lang_uk    Ukrainian
lr=lang_vi    Vietnamese

**Regards** :octocat:

**Comment by qinyuanqiblog** [comment]
@spmbt Thank you for taking my advice. Thanks a million.

**Comment by spmbt** [comment]
mikhoul: Full list for 2018-01-23:
https://www.google.com/preferences?hl=en#languages (was shown 4 columns)

de: Deutsch
en: English
es: español
es-419: español (Latinoamérica)
fr: français
hr: hrvatski
it: italiano
nl: Nederlands
pl: polski
pt-BR: português (Brasil)
pt-PT: português (Portugal)
vi: Tiếng Việt
tr: Türkçe
ru: русский
ar: العربية
th: ไทย
ko: 한국어
zh-CN: 中文 (简体)
zh-TW: 中文 (繁體)
ja: 日本語

Show more:
----------------
ach: Acoli
af: Afrikaans
ak: Akan
az: azərbaycan
ban: Balinese
xx-bork: Bork, bork, bork!
bs: bosanski
br: brezhoneg
ca: català
ceb: Cebuano
cs: čeština
sn: chiShona
co: Corsican
cy: Cymraeg
da: dansk
yo: Èdè Yorùbá
et: eesti
xx-elmer: Elmer Fudd
eo: esperanto
eu: euskara
ee: Eʋegbe
tl: Filipino
fo: føroyskt
fy: Frysk
gaa: Ga
ga: Gaeilge
gd: Gàidhlig
gl: galego
gn: Guarani
xx-hacker: Hacker
ht: Haitian Creole
ha: Hausa
haw: ʻŌlelo Hawaiʻi
bem: Ichibemba
ig: Igbo
rn: Ikirundi
id: Indonesia
ia: Interlingua
zu: isiZulu
is: íslenska
jw: Javanese
rw: Kinyarwanda
sw: Kiswahili
tlh: Klingon
kg: Kongo
mfe: kreol morisien
kri: Krio (Sierra Leone)
la: Latin
lv: latviešu
to: lea fakatonga
lt: lietuvių
ln: lingála
loz: Lozi
lua: Luba-Lulua
lg: Luganda
hu: magyar
mg: Malagasy
mt: Malti
mi: Maori
ms: Melayu
pcm: Nigerian Pidgin
no: norsk
nso: Northern Sotho
ny: Nyanja
nn: nynorsk
uz: o‘zbek
oc: Occitan
om: Oromoo
xx-pirate: Pirate
pt: português
ro: română
mo: română (Republica Moldova)
rm: rumantsch
qu: Runasimi
nyn: Runyankore
crs: Seychellois Creole
sq: shqip
sd: Sindhi
sk: slovenčina
sl: slovenščina
so: Soomaali
st: Southern Sotho
sr-ME: srpski (Crna Gora)
sr-Latn: srpski (latinica)
su: Sundanese
fi: suomi
sv: svenska
tn: Tswana
tum: Tumbuka
tk: Turkmen
tw: Twi
wo: Wolof
xh: Xhosa
el: Ελληνικά
be: беларуская
bg: български
ky: кыргызча
kk: қазақ тілі
mk: македонски
mn: монгол
sr: српски
tt: татар
tg: тоҷикӣ
uk: українська
ka: ქართული
hy: հայերեն
yi: ייִדיש
iw: עברית
ug: ئۇيغۇرچە
ur: اردو
ps: پښتو
fa: فارسی
ckb: کوردیی ناوەندی
ti: ትግርኛ
am: አማርኛ
ne: नेपाली
mr: मराठी
hi: हिन्दी
bn: বাংলা
pa: ਪੰਜਾਬੀ
gu: ગુજરાતી
or: ଓଡ଼ିଆ
ta: தமிழ்
te: తెలుగు
kn: ಕನ್ನಡ
ml: മലയാളം
si: සිංහල
lo: ລາວ
my: မြန်မာ
km: ខ្មែរ
chr: ᏣᎳᎩ
